native-embedding/README.md

```diff
@@ -0,0 +1,20 @@
+# Polyglot Native Embedding Demo with GraalVM for JDK 21 
+Demonstration project showing polyglot native embedding with GraalVM JDK 21 using Maven.
+It contains a simple API class that is built into a native libary and then accessed in a C program.
+The C program runs both with the JVM library and the self-contained library built by Native Image
+as a part of this project. 
+
+## Setup
+
+[Download](https://www.graalvm.org/downloads/) GraalVM and point the `JAVA_HOME` environment variable to it.
+
+## Maven Usage
+
+Download Maven or import as Maven project into your IDE.
+
+* `mvn verify` to run the C tests of the API using the JVM library.
+* `mvn -Pnative-embedding-library package` to build the native embedding library by Native Image.
```

Comment:
When doing this, I'm getting quite a few warnings about deprecated and experimental options from native-image. This is an example people will copy from, I think we shouldn't use any of these. If absolutely necessary, we should at least unlock the experimental options, but better would be to provide an example that needs only stable options.

```
Warning: Using a deprecated option --rerun-class-initialization-at-runtime= from 'META-INF/native-image/org.graalvm.shadowed.icu4j/native-image.properties' in 'file:///home/roland/.m2/repository/org/graalvm/shadowed/icu4j/23.1.0/icu4j-23.1.0.jar'. Currently there is no replacement for this option. Try using --initialize-at-run-time or use the non-API option -H:ClassInitialization directly.
Warning: The option '-H:Path=/home/roland/git/misc/polyglot-embedding-demo/target' is experimental and must be enabled via '-H:+UnlockExperimentalVMOptions' in the future.
Warning: The option '-H:Name=embedding' is experimental and must be enabled via '-H:+UnlockExperimentalVMOptions' in the future.
Warning: Please re-evaluate whether any experimental option is required, and either remove or unlock it. The build output lists all active experimental options, including where they come from and possible alternatives. If you think an experimental option should be considered as stable, please file an issue.
```

Reply:
This PR is adding a completely independent maven project to a sub-directory native-embedding ( I will add information about this to the main README file), the list of warnings therefore does not correspond to the added project, the correct list is:
```
Warning: Using a deprecated option --rerun-class-initialization-at-runtime= from 'META-INF/native-image/org.graalvm.shadowed.icu4j/native-image.properties' in 'file:///home/jchaloup/.m2/repository/org/graalvm/shadowed/icu4j/23.1.0/icu4j-23.1.0.jar'. Currently there is no replacement for this option. Try using --initialize-at-run-time or use the non-API option -H:ClassInitialization directly.
```
And I can't influence it from this project as it is imported dependency.